src/runner-worker.ts

```diff
@@ -114,9 +114,17 @@ async function doWork(prefix: string, queue: ITestRunFile[]) {
       for (const include of next.include || []) {
         args.push("--test-name-pattern", `^${escapeRegex(include)}$`);
       }
+
+      if (next.path.endsWith("ts") || next.path.endsWith("tsx")) {
+        args.push("--loader");
+        args.push("tsx");
+      }
```

Comment:
This needs to be a setting, it should not be hardcoded to use any specific loader

Reply:
See below.